### PR TITLE
Hoge Finance Ethereum To Base Bridge ($HOGE)

### PR DESCRIPTION
**Pull request by blasthoge**
This is a bridge for the Hoge Finance ($HOGE) token. This bridge was deployed by community member Kash Corleone via Base Migrate.

Learn more about Hoge Finance: 

Official Website: https://hogefinance.net/

Token Report: https://hoge.report 

Learn more about Base Migrate: https://www.base-migrate.xyz/

IMG_6307

HOGE is now BASED 👍

### CODE REVIEW

**Comment by blasthoge** [comment]
wen merge

**Comment by cfluke-cb** [comment]
Hi @blasthoge Can you update the website to a uri format? 
error: HOGE: instance.website: does not conform to the "uri" format
www.Hoge.report => https://www.hoge.report

**Comment by wbnns** [comment]
@blasthoge 

Since it's been a while and we haven't heard back, recommending to close this for now. Please let us know if you'd still like to pursue this addition and we can reopen. 👍

**Comment by blasthoge** [comment]
HI there!

Would love to reopen and finish this process. Updating to URI format now.

Let me know if any follow-up steps are needed.

Best,

File: data/HOGE/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hoge.finance",
+  "symbol": "HOGE",
+  "decimals": 9,
+  "description": "Hoge Finance ($HOGE) is a deflationary ERC720 token featuring a 2% burn and 1% reflection mechanism on every transaction.\n\nOriginally deployed on the Ethereum L1 network.",
+  "website": "www.Hoge.report",
```
Comment (wbnns) [inline data/HOGE/data.json:6]: @blasthoge 

Please update this as per @cfluke-cb 's request and also the other links to match 🙏

**Review by wbnns — CHANGES_REQUESTED** [review]
@blasthoge 

Hello, I left a comment with an example of the URI that needs to be added.

**Comment by blasthoge** [comment]
Will do!

File: data/HOGE/data.json (hunk 1 above)
Comment (blasthoge) [reply data/HOGE/data.json:6]: on it now

**Review by blasthoge — COMMENTED** [review]
Just added this!

**Review by wbnns — CHANGES_REQUESTED** [review]
@blasthoge 

Thanks for the update! Still seeing this error:
```
error: HOGE: instance.website: does not conform to the "uri" format
```
Your website should include the `https://`

Also, it looks like the SVG image isn't rendering. Can you please fix that as well?

**Comment by wbnns** [comment]
@blasthoge Since it's been a while and we haven't heard back, closing this for now. Please let us know if you'd like to pursue the additional changes to resolve the outstanding issues on the PR and we can reopen. 👍

**Pull request closed without merging**